components/base/sensorcontrolled/velocities.go

```diff
@@ -30,12 +30,12 @@ func (sb *sensorBase) SetVelocity(
 			return err
 		}
 	}
-	sb.loop.Resume()
 
 	// convert linear.Y mmPerSec to mPerSec, angular.Z is degPerSec
 	if err := sb.updateControlConfig(ctx, linear.Y/1000.0, angular.Z); err != nil {
 		return err
 	}
+	sb.loop.Resume()
```

Comment:
does this cause a pause between set velocity calls in the same direction at different speeds? for example, set velocity at 100 mm/s and then set velocity at 200 mm/s, do the wheels stop before going to 200 or just speed up?

Reply:
nvm I see the changes in resume